src/main/groovy/nl/javadude/gradle/plugins/license/LicenseResolver.groovy

```diff
@@ -134,36 +143,77 @@ class LicenseResolver {
     /**
      * Provide full list of resolved artifacts to handle for a given project.
      *
-     * @param project                       the project
+     * @param project the project
      * @return Set with resolved artifacts
      */
     Set<ResolvedArtifact> resolveProjectDependencies(Project project) {
-
         Set<ResolvedArtifact> dependenciesToHandle = new HashSet<ResolvedArtifact>()
-        def subprojects = project.rootProject.subprojects.groupBy { Project p -> "$p.group:$p.name:$p.version".toString()}
 
-        if (project.configurations.any { it.name == dependencyConfiguration && isResolvable(it) }) {
+        if (project.configurations.any { it.name == dependencyConfiguration }) {
             def configuration = project.configurations.getByName(dependencyConfiguration)
-            configuration.resolvedConfiguration.resolvedArtifacts.each { ResolvedArtifact d ->
-                String dependencyDesc = "$d.moduleVersion.id.group:$d.moduleVersion.id.name:$d.moduleVersion.id.version".toString()
-                if(isDependencyIncluded(dependencyDesc)) {
```

Comment:
This line does not work in new code, so it's not possible to exclude any project dependency